File: 1. Iniciante/URI1037.cpp
#include <iostream>
#include <cstdio>

using namespace std;

int main() {
    double num;

    cin >> num;

    if (num >= 0.00 && num <=25.00){
        printf("Intervalo [0,25]\n");
    } else if ( num > 25.00 && num <= 50.00){
        printf("Intervalo (25,50]\n");
    } else if ( num > 50.00 && num <= 75.00){
        printf("Intervalo (50,75]\n");
    } else if ( num > 75.00 && num <= 100.00){
        printf("Intervalo (75,100]\n");
    } else{
            printf("Fora de intervalo\n");
    }
    return 0;
}

File: 1. Beginner/URI1037.py
num = float(input(""))

if num >= 0.00 and num <=25.00:
    print("Intervalo [0,25]")
elif num > 25.00 and num <= 50.00:
    print("Intervalo (25,50]")
elif num > 50.00 and num <= 75.00:
    print("Intervalo (50,75]")
elif num > 75.00 and num <= 100.00:
    print("Intervalo (75,100]")
else:
    print("Fora de intervalo")

